### PR TITLE
Added push of container images to quay

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
I tested this by running it locally and it works. This is the new output lines:

```
Performing a 'docker login' for Quay
WARNING! Your password will be stored unencrypted in /home/jpkroehling/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded
Tagging jaegertracing/jaeger-operator:master as quay.io/jaegertracing/jaeger-operator:master
Pushing 'quay.io/jaegertracing/jaeger-operator:master'
The push refers to repository [quay.io/jaegertracing/jaeger-operator]
684b22d2ecb9: Pushed 
7722e09c2f35: Pushed 
30d9f2e92f64: Pushed 
6e50279be129: Layer already exists 
b36cd6355d69: Layer already exists 
d69483a6face: Layer already exists 
master: digest: sha256:49c1de0590c723f26bd8945a0ba4b4fee794b51d1d187c976c16cfb6d66a9139 size: 7541
```

And it shows up in quay as expected:
image

File: .travis/publish-images.sh [hunk 1]
```diff
@@ -25,4 +25,22 @@ if [ "${MAJOR_MINOR}x" != "x" ]; then
     MAJOR_MINOR_IMAGE="${BASE_BUILD_IMAGE}:${MAJOR_MINOR}"
     docker tag "${BUILD_IMAGE}" "${MAJOR_MINOR_IMAGE}"
     docker push "${MAJOR_MINOR_IMAGE}"
-fi
\ No newline at end of file
+fi
+
+## now, push to quay.io
+if [ "${QUAY_PASSWORD}x" != "x" -a "${QUAY_USERNAME}x" != "x" ]; then
```
Comment (pavolloffay) [inline .travis/publish-images.sh]: nice trick? 

Is it better than testing on empty env var?

Comment (jpkrohling) [reply .travis/publish-images.sh]: I guess I'm dating myself. There's probably a better (modern) way to do that, but old shells (in some Solaris versions, IIRC) didn't handle string comparisons correctly. I guess it's just one of those things that we get used to and never change :)

Comment (jpkrohling) [reply .travis/publish-images.sh]: @mpersano taught me that trick, perhaps he remembers what went wrong, where and why :)

**Pull request merged**